stories/Button.jsx

```diff
@@ -1,21 +1,53 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import './button.css';
+import React from "react";
+import PropTypes from "prop-types";
+import "./button.css";
+import styled, { keyframes } from "styled-components";
+
+const shake = keyframes`
```

Comment:
really like the use for the keyframes animation to the buttons. nicely presentable 👍